oshi-core/src/test/java/oshi/util/ParseUtilTest.java

```diff
@@ -450,8 +450,8 @@ public void testParsePnPDeviceIdToVendorProductId() {
         Pair<String, String> idPair = ParseUtil
                 .parsePnPDeviceIdToVendorProductId("PCI\\VEN_10DE&DEV_134B&SUBSYS_00081414&REV_A2\\4&25BACB6&0&00E0");
         assertNotNull(idPair);
-        assertEquals("0x10de", idPair.getA());
-        assertEquals("0x134b", idPair.getB());
+        assertEquals("First element of pair","0x10de", idPair.getA());
```

Comment:
Can you make this sentence stand alone, like "First element of pair mismatch."  Also leave a space after the comma so checkstyle doesn't complain. :)

Reply:
addressed.